docs-v2/integrations/all/personio-v2.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Personio (v2)
+sidebarTitle: Personio (v2)
+---
+
+API configurations: [`personio`](https://nango.dev/providers.yaml)
```

Comment:
I think you can remove this page and just add `personio-v2` in personio.mdx

Reply:
Hey @bodinsamuel I just wonder if we can merge both since they are slightly different:

personio:
    display_name: Personio
    categories:
        - hr
    auth_mode: OAUTH2_CC
    proxy:
        base_url: https://api.personio.de/v1
        headers:
            x-personio-partner-id: ${connectionConfig.partnerId}
            x-personio-app-id: ${connectionConfig.appId}
    token_url: https://api.personio.de/v1/auth
    body_format: json
    docs: https://docs.nango.dev/integrations/all/personio
    connection_config:
        partnerId:
            type: string
            title: Partner ID
            description: The partner ID of your Personio account
        appId:
            type: string
            title: App ID
            description: The app ID of your Personio account

personio-v2:
    display_name: Personio (v2)
    categories:
        - hr
    auth_mode: OAUTH2_CC
    proxy:
        base_url: https://api.personio.de/v2
        headers:
            Content-Type: application/x-www-form-urlencoded
    token_url: https://api.personio.de/v2/auth/token
    scope_separator: ' '
    docs: https://docs.nango.dev/integrations/all/personio
    token_params:
        grant_type: client_credentials

Reply:
Good point. Maybe we should create a v2 section in the same doc.

Reply:
Do you have any example of an integration where multiple versions are explained so I can make sure we are being consistent on how things are shown?

Reply:
We never did it so I guess something simple like

```
## API v2
[... requirements]
```

Reply:
Hey @bodinsamuel actually there were no big differences between both versions from the nango perspective. What I have done is adding a `Api V1` tag to the requirements that are specific for that version